compat/test/browser/forwardRef.test.js

```diff
@@ -400,8 +400,9 @@ describe('forwardRef', () => {
 		const Transition = ({ children }) => {
 			const state = useState(0);
 			forceTransition = state[1];
-			expect(children.ref).to.not.be.undefined;
-			if (state[0] === 0) expect(children.props.ref).to.be.undefined;
+			// TODO: ref exists on the backing node
+			// expect(children.ref).to.not.be.undefined;
```

Comment:
We can't really test this anymore since ref and props.ref are at the backing node level now